src/core/caniuse.js

```diff
@@ -0,0 +1,289 @@
+/*
+Module: "core/caniuse"
+Adds a caniuse support table for a "feature" #1238
+
+`conf.caniuse = { feature: key }` =>
+  .. the table is added before Copyright
+`<section data-caniuse=key>` =>
+  .. table will be added after section's first element (usually section heading)
+Optional settings:
+  `conf.caniuse.browsers` list of caniuse supported browser names
+    to be shown in the table or "ALL"
+    default: ["chrome", "firefox", "safari", "edge"]
+  `conf.caniuse.versions` number of older browser versions to show
+  `conf.caniuse.maxAge` (in ms) local response cache duration
+*/
+
+import { semverCompare } from "core/utils";
+import { pub } from "core/pubsubhub";
+import "deps/hyperhtml";
+import caniuseCss from "deps/text!core/css/caniuse.css";
+
+// Opportunistically insert the style into the head to reduce FOUC.
+const codeStyle = document.createElement("style");
+codeStyle.textContent = caniuseCss;
+document.head.appendChild(codeStyle);
+
+export const name = "core/caniuse";
+
+export function run(conf) {
+  normalizeConf(conf);
```

Comment:
We should bail out early here. Adding the defaults when they won't do anything doesn't seem great. 

```
if(!conf.caniuse){
   return; // nothing to do
}
normalizeConf(conf);
```

Reply:
I did this because we have an option to add caniuse by `<section data-caniuse>` which doesn't require `conf.caniuse` to declared.